src/main.rs

```diff
@@ -12,8 +16,39 @@ use xtensa_atomic_emulation_trap as _;
 use xtensa_lx_rt::entry;
 {%- endif %}
 
+{% if alloc -%}
+#[global_allocator]
+static ALLOCATOR: esp_alloc::EspHeap = esp_alloc::EspHeap::empty();
+
+fn init_heap() {
+    const HEAP_SIZE: usize = 32 * 1024;
```

Comment:
Is this a reasonable default value?

Reply:
I think we can start with this and maybe add a comment for the user

The best solution would be to add the heap_end symbol for ESP32-C3 and use the values provided by the linker script, certainly